src/components/SignoffSummary/index.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Fragment } from 'react';
+import { makeStyles } from '@material-ui/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import Typography from '@material-ui/core/Typography';
+import { object } from 'prop-types';
+
+const useStyles = makeStyles(theme => ({
+  listSubheader: {
+    lineHeight: 1.5,
+    marginBottom: theme.spacing(1),
```

Comment:
`marginBottom: theme.spacing(0.5),`

Also let add `paddingLeft: theme.spacing(1),`.

Reply:
Thanks for this, I see that it cleans up the alignment!